packages/admin/admin/src/index.ts

```diff
@@ -108,6 +108,7 @@ export { MuiThemeProvider } from "./mui/ThemeProvider";
 export { RouterBrowserRouter } from "./router/BrowserRouter";
 export { RouterConfirmationDialog, RouterConfirmationDialogClassKey, RouterConfirmationDialogProps } from "./router/ConfirmationDialog";
 export { RouterContext } from "./router/Context";
+export { ForcePromptRoute } from "./router/ForcePromptRoute";
```

Comment:
Should we even export this? Could there be other use cases than StackSwitch (which does it automatically)?

Reply:
Although I don't have a use case in mind I would export this.